src/router.ts

```diff
@@ -416,8 +416,10 @@ export async function handleNewVoter(
       userOptions.userPhoneNumber
     );
     if (knownState) {
+      // the knownState value from the db is probably a state abbreviation, so we need to resolve it
```

Comment:
nit: Can you please capitalize the first first word of each of these comments to be consistent with the rest of the codebase?

Reply:
fixed!